app/src/main/res/layout/layout_nav_header.xml

```diff
@@ -31,10 +32,12 @@
         android:typeface="monospace"/>
 
     <TextView
-        android:id="@+id/textView"
+        android:id="@+id/nav_user_tenant"
         android:layout_height="wrap_content"
         android:layout_width="wrap_content"
         android:textColor="@color/white"
-        android:text="rajanmaurya154@gmail.com"/>
+        android:paddingTop="@dimen/layout_padding_8dp"
```

Comment:
Adding a top padding will look good at latest mobile with full screens as there , we see a overflow at the top of nav bar but it works fine in the older mobiles where we used to have touch screen buttons. For solving this issue , dont add padding, I have already opened a [PR](https://github.com/apache/fineract-cn-mobile/pull/118) that solves this issue.

Reply:
I didnt got it. Should i remove top padding or increase it? I have not faced and problem of overflow.

Reply:
I added it because it wasn't looking good two textview sticked together also they have same text size so it would look confusing.

Reply:
I reduced 8 dp from image view and added 8dp in textview so dont worry it wont go out of 180dp as is has not changed total.